app/services/analytics_events.rb

```diff
@@ -84,6 +84,13 @@ def account_visit
     track_event('Account Page Visited')
   end
 
+  # @identity.idp.event_name Add Email: Email Confirmation
+  # @param [Hash] result results from an email confirmation
```

Comment:
now there is no param named `result` ... can we document any keys we know of for this hash?

Reply:
Check now?

Reply:
Looks like the callsite was missing `**`, so I added that in f1410337db4d57e5751d5861e894bdc1880dfd23

Also when inspecting locally, found a `success:` key, so I added that in there too